### PR TITLE
Add initial repo template based on https://github.com/ni/Github-Repo-Template

### PR DESCRIPTION
**Pull request by pvallone**
[X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/systemlink-OpenAPI-documents/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?
Adds a minimal GitHub repository template, which is expected of all repos hosted under github.com/ni.

@jjaco2 : feel free to just review the stuff in README.md, as that's essentially user-facing documentation. I consider all of that to be pretty rough, and I imagine we'll just replace it with links to whatever Swagger documentation we end up creating on ni.com.

### Why should this Pull Request be merged?
A repo template is expected to be present in all repos hosted under github.com/ni.

### What testing has been done?
N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Contributing to systemlink-OpenAPI-documents 
+
+Contributions to systemlink-OpenAPI-documents are welcome from all!
+
+systemlink-OpenAPI-documents is managed via [git](https://git-scm.com), with the canonical upstream
+repository hosted on [GitHub](http://developercertificate.org/).
```
Comment (spanglerco) [inline CONTRIBUTING.md]: Is this meant to be a link to GitHub instead of developercertificate.org?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# SystemLink OpenAPI Documents
+
+This repository contains OpenAPI documents which describe some of the HTTP APIs exposed by web services in SystemLink and SystemLink Cloud.
+
+## Using an OpenAPI document to generate a client
+
+To generate a client from an OpenAPI document, paste the document into editor.swagger.io, click "Generate Client", and select a language.
```
Comment (spanglerco) [inline README.md]: You could make editor.swagger.io a link.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+systemlink-OpenAPI-documents follows a pull-request model for development.  If you wish to
+contribute, you will need to create a GitHub account, fork this project, push a
+branch with your changes to your project, and then submit a pull request.
+
+See [GitHub's official documentation](https://help.github.com/articles/using-pull-requests/) for more details.
+
+# Testing
+
+Before making a pull request, ensure that there are no syntax errors in your document by pasting it into editor.swagger.io.
```
Comment (spanglerco) [inline CONTRIBUTING.md]: You could make editor.swagger.io a link.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+       person who certified (a), (b) or (c) and I have not modified
+       it.
+
+   (d) I understand and agree that this project and the contribution
+       are public and that a record of the contribution (including all
+       personal information I submit with it, including my sign-off) is
+       maintained indefinitely and may be redistributed consistent with
+       this project or the open source license(s) involved.
+
+(taken from [developercertificate.org](http://developercertificate.org/))
```
Comment (spanglerco) [inline CONTRIBUTING.md]: I lean towards using https links when available.

**Review by spanglerco — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (pvallone) [reply CONTRIBUTING.md]: Yeah, I think so. That came from the NI template, so I'll try to get it fixed there as well.

**Review by spanglerco — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (pvallone) [reply CONTRIBUTING.md]: The template change was upstreamed, by the way: https://github.com/ni/Github-Repo-Template/pull/2

File: README.md (hunk 2 above)
Comment (jjaco2) [reply README.md]: > HTTP APIs exposed by web services

"HTTP APIs" and "web services" might seem synonymous to users, so either cut out "web services" or change "exposed by" to something more specific. Do web services enable the HTTP APIs, or include them, or anything that might clarify that relationship?

**Review by jjaco2 — COMMENTED** [review]
Minor suggestion to clarify relationship between HTTP APIs and SystemLink/SLC.

**Review by jjaco2 — APPROVED** [review]

**Pull request merged**